locustfile.py

```diff
@@ -192,8 +192,14 @@ def poll_upload_until_ready(self, url):
             with self.client.get(url, allow_redirects=False,
                                  name='/en-US/developers/upload/:uuid/json',
                                  catch_response=True) as response:
-                if response.status_code == 200:
+                try:
                     data = response.json()
+                except ValueError:
+                    return response.failure(
```

Comment:
I wanted to know the status code (maybe 404) and the content it choked on. Since I really don't know what case this happens in.